docs/src/reference/the-traversal.asciidoc

```diff
@@ -2309,6 +2309,25 @@ link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gre
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.html#limit(org.apache.tinkerpop.gremlin.process.traversal.Scope,long)++[`limit(Scope,long)`]
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/Scope.html++[`Scope`]
 
+[[list-none-step]]
+=== List None Step
```

Comment:
Users will still see this as `none()` so I think it's better to explain the two usages in the existing section for that step. Luckily `none()` is mostly an internal step. it's unlikely anyone is really using it directly.